MyGame/MyGame/GameScene.swift

```diff
@@ -0,0 +1,113 @@
+//
+//  GameScene.swift
+//  MyGame
+//
+//  Created by Sacchitiello Fabio on 2/28/18.
+//  Copyright © 2018 Sacchitiello Fabio. All rights reserved.
+//
+
+import SpriteKit
+import GameplayKit
+
+class GameScene: SKScene, SKPhysicsContactDelegate {
+    
+    //store score and display labels
+    var isGameStarted = Bool (false)
+    var isDead = Bool (false)
+    let collectSound = SKAction.playSoundFileNamed("CoinSound.mp3", waitForCompletion: false)
+    
+    var score = Int(0)
+    
+    var scoreLabel = SKLabelNode()
+    var highscoreLabel = SKLabelNode()
+    var playLabel = SKLabelNode()
+    
+    var restartButton = SKSpriteNode()
+    var pauseButton = SKSpriteNode()
+    var logoImg = SKSpriteNode()
+    
+    var wallBlocks = SKNode()
+    var moveAndRemove = SKAction()
+    
+    //Chopper atlas for animation
+    //let chopperAtlas = SKTextureAtlas(named: "player")
+    var chopperSprites: [SKTexture] = [SKTexture(imageNamed: "img1"), SKTexture(imageNamed: "img2"), SKTexture(imageNamed: "img3"), SKTexture(imageNamed: "img4")]
```

Comment:
because you used an extension you have to place the choppers textures in the game scene, but should game scenes be managing textures? or should they focus just on the game management?